flags/flags.go

```diff
@@ -7,21 +7,17 @@ import (
 
 var (
 	// Flags for running the Chainbridge app
-	ChainConfigFlagName   = "config"
-	RelayerConfigFlagName = "relayer-config"
-	KeystoreFlagName      = "keystore"
-	BlockstoreFlagName    = "blockstore"
-	FreshStartFlagName    = "fresh"
-	LatestBlockFlagName   = "latest"
-	TestKeyFlagName       = "testkey"
+	ConfigFlagName      = "config"
```

Comment:
How do you think about renaming variable and value to something like ConfigPathFlagName = "cfgPath" or something like that. Just found it unclear

Reply:
Definitely, we can rename Flag to ConfigPathFlagName to keep it more simple for us, but I checked some CLIs and --config is pretty standard (wget, curl, docker).